src/vue/common/fees/fees.mixin.js

```diff
@@ -0,0 +1,37 @@
+import { PAYMENT_FEE_SUBTYPES } from '@tokend/js-sdk'
+import Fees from './Fees'
+import { Api } from '@/api'
+
+export default {
+  components: {
+    Fees,
+  },
+  methods: {
+    async calculateFees (opts) {
+      const requestUrl = `v3/accounts/${opts.accountId}/calculated_fees`
```

Comment:
1. Add two methods for "Recipient Fees" and for "Sender Fees" => (get two ids)
2. Add "Both Calculate Fees" => (use two ids)